regression/cbmc-java/stack_var4/test.desc

```diff
@@ -3,8 +3,6 @@ stack_test.class
 
 ^EXIT=0$
 ^SIGNAL=0$
-^.*assertion at file stack_test.java line 5 function.*: SUCCESS
```

Comment:
When the simplifier gives a SUCCESSFUL verdict you don't get the specific goal printouts. Perhaps that ought to change, but I'd like to do that elsewhere if so. I'm unsure exactly how this now passes the simplifier and didn't before (suspect: typecasts, which are now less likely to occur nested?) However a look over the GOTO programs for the tests shows they're still testing what they intended to, so I don't think the tests are any more or less trivial than they were before.